src/components/VaultSummary.tsx

```diff
@@ -53,22 +59,42 @@ type Props = {
 };
 
 const VaultSummary = ({ vaultKey }: Props) => {
-  const { vaults, errors } = useVaultStore(state => ({
+  const {
+    vaults,
+    errors,
+    prices,
+    priceErrors,
+    vaultMetrics,
+    vaultGovernedParams,
+    vaultManagerLoadingErrors,
+    managers,
+  } = useVaultStore(state => ({
     vaults: state.vaults,
+    vaultMetrics: state.vaultMetrics,
+    vaultGovernedParams: state.vaultGovernedParams,
     errors: state.vaultErrors,
+    prices: state.prices,
+    priceErrors: state.priceErrors,
+    vaultManagerLoadingErrors: state.vaultManagerLoadingErrors,
+    managers: state.vaultManagers,
   }));
+
   const vault = vaults?.get(vaultKey);
-  const error = errors.get(vaultKey);
+  assert(vault, `Cannot render summary for nonexistent vault ${vaultKey}`);
+
   const displayFunctions = useAtomValue(displayFunctionsAtom);
 
-  return useMemo(() => {
-    assert(vault, `Cannot render summary for nonexistent vault ${vaultKey}`);
-    assert(
-      displayFunctions,
-      `Cannot render summary for vault ${vaultKey} - missing vbank asset info.`,
-    );
-    const { displayBrandPetname, displayBrandIcon } = displayFunctions;
+  const metrics = vaultMetrics?.get(vault?.managerId ?? '');
+  const params = vaultGovernedParams?.get(vault?.managerId ?? '');
+  const brand = metrics?.totalCollateral?.brand;
+  const price = brand && prices.get(brand);
+  const manager = managers.get(vault?.managerId ?? '');
```

Comment:
can that store have '' as a key?

Reply:
Nope

Reply:
so why pass it in to `get`?
```suggestion
  const manager = vault?.managerId && managers.get(vault.managerId);
```

Reply:
I just thought it was more succinct to cast to a string inside the arg